Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionMapsView.cs

```diff
@@ -133,6 +135,12 @@ private void OnKeyDownEventForDelete()
             item.DeleteItem();
         }
 
+        private void OnKeyDownEventForDuplicate()
+        {
+            var item = (InputActionMapsTreeViewItem)m_ListView.GetRootElementForIndex(m_ListView.selectedIndex);
```

Comment:
Necessary to store item var here?

Reply:
no!  I just did that to make it more readable. I combined the 2 lines into one now.